src/CppParser/Bindings/CSharp/i686-pc-win32-msvc/CppSharp.CppParser.cs

```diff
@@ -700,7 +700,7 @@ public new partial struct __Internal
                     [SuppressUnmanagedCodeSecurity]
                     [DllImport("CppSharp.CppParser.dll", CallingConvention = global::System.Runtime.InteropServices.CallingConvention.ThisCall,
                         EntryPoint="??0TagType@AST@CppParser@CppSharp@@QAE@ABV0123@@Z")]
-                    internal static extern global::System.IntPtr cctor_1(global::System.IntPtr instance, global::System.IntPtr _0);
+                    internal static extern global::System.IntPtr cctor_2(global::System.IntPtr instance, global::System.IntPtr _0);
```

Comment:
It's most probably unrelated but please do me a favour and debug why you got all of these cctor_2 instead of cctor_1.